debezium-sink/src/main/java/io/debezium/bindings/kafka/DebeziumSinkRecord.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.bindings.kafka;
+
+import org.apache.kafka.connect.data.Struct;
+
+public interface DebeziumSinkRecord extends io.debezium.sink.DebeziumSinkRecord {
```

Comment:
Could we add some documentation here, given it's an interface?

Reply:
Done